### PR TITLE
add ability to set custom "Undefined" status key value to `Mpd2Widget`

### PR DESCRIPTION
**Pull request by mrtnvgr**
This PR implements "undefined_value" option to `Mpd2Widget`.

Since it is very opinionated, we can allow user to change this value (like a `idle_message`).

_In my opinion, "Undefined" looks bad in the status bar, changing it to something like "[-]" would be better.
I see it regularly, because a lot of my music doesn't have any tags._

### CODE REVIEW

File: test/widgets/test_mpd2widget.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import sys
+from collections import defaultdict
```
Comment (elParaguayo) [inline test/widgets/test_mpd2widget.py:21]: Why is this needed? It doesn't look like your test calls this.

**Review by elParaguayo — COMMENTED** [review]
Thanks for this. Looks ok to me other than one minor comment.

File: test/widgets/test_mpd2widget.py (hunk 1 above)
Comment (mrtnvgr) [reply test/widgets/test_mpd2widget.py:21]: > Why is this needed? It doesn't look like your test calls this.

Oh, my bad! I was testing some stuff and forgot to remove this.

**Comment by mrtnvgr** [comment]
@elParaguayo
